npc/controller_test.go

```diff
@@ -79,7 +79,11 @@ func (i *mockIPSet) entryExists(ipsetName ipset.Name, entry string) bool {
 }
 
 func (i *mockIPSet) Flush(ipsetName ipset.Name) error {
-	return errors.New("Not Implemented")
+	if _, ok := i.sets[string(ipsetName)]; !ok {
```

Comment:
This change is not required for your PR. If you want to keep it, then instead of deleting the given ipset, you should remove its entries.

Reply:
@brb removed it, will add it when needed.